content/en/observability_pipelines/destinations/_index.md

```diff
@@ -29,7 +29,43 @@ Select and set up your destinations when you [set up a pipeline][2]. This is ste
     {{< nextlink href="observability_pipelines/destinations/#rsyslog-or-syslog-ng" >}}Rsyslog or Syslog-ng{{< /nextlink >}}
     {{< nextlink href="observability_pipelines/destinations/#splunk-http-event-collector-hec" >}}Splunk HTTP Event Collector (HEC){{< /nextlink >}}
     {{< nextlink href="observability_pipelines/destinations/#sumo-logic-hosted-collector" >}}Sumo Logic Hosted Collector{{< /nextlink >}}
-{{< /whatsnext >}}
+{{< /whatsnext >}} 
+
+### Batch settings {#batch-settings-intro}
+
+Observability Pipelines destinations batch events sent to the downstream integration. The following parameters define each destinations' batch settings:
+
+- Maximum number of events
+- Maximum number of bytes
+- Timeout (seconds)
+
+When one of these parameters is met, the batch of events is flushed. For example, if a destination's parameters are:
+
+- Maximum number of events = 2
+- Maximum number of bytes = 10,0000
+- Timeout (seconds) = 5
+
+And the destination receives 1 event in a 5-second window, it flushes the batch at the 5-second timeout.
+
+If the destination receives 3 events within 2 seconds, it flushes a batch with 2 events and then flushes a second batch with the remaining event after 5 seconds. If the destination receives 1 event that is more than 10,0000 bytes, it flushes this batch with the 1 event.
+
+These are the batch settings for each destination:
+
+| Destination                                | Max Events     | Max Bytes       | Timeout (seconds)   |
+|--------------------------------------------|----------------|-----------------|---------------------|
+| Amazon OpenSearch                          | None           | 10,000,000      | 1                   |
+| Amazon S3 (Datadog Log Archives)           | None           | 100,000,000     | 900                 |
+| Azure Storage (Datadog Log Archives)       | None           | 100,000,000     | 900                 |
+| Datadog Logs                               | 1,000          | 4,250,000       | 5                   |
+| Datadog Metrics                            | 100,000        | None            | 2                   |
```

Comment:
We should omit this one for now because it isn't released yet. But I guess we'll need to periodically come back to this table to add others as we add destinations.

Reply:
Ah yes, good point. Removed, but noted in the docs draft for to add batch settings.